sdk/search/Microsoft.Azure.Search.Data/src/Microsoft.Azure.Search.Data.csproj

```diff
@@ -4,16 +4,9 @@
     <Description>Use this package if you're developing a .NET application using Azure Search, and you only need to query or update documents in your indexes. If you also need to create or update indexes, synonym maps, or other service-level resources, use the Microsoft.Azure.Search package instead.</Description>
     <AssemblyTitle>Microsoft Azure Search Data Library</AssemblyTitle>
     <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
-    <VersionPrefix>9.0.2</VersionPrefix>
     <PackageReleaseNotes>See the Microsoft.Azure.Search package for detailed release notes on the entire Azure Search .NET SDK.</PackageReleaseNotes>
   </PropertyGroup>
-
-  <PropertyGroup Condition=" '$(Configuration)' == 'Debug' ">
-    <!--There are certain projects that needs internals visible for test purpose -->
-    <SignAssembly>true</SignAssembly>
-    <AssemblyOriginatorKeyFile>$(RepoEngPath)\MSSharedLibKey.snk</AssemblyOriginatorKeyFile>
-  </PropertyGroup>
-
+  <Import Project="..\..\Microsoft.Azure.Search.Version.props" />
```

Comment:
You can manually import this if you like or you can name it Directory.Build.props and have it automatically get imported by msbuild (of course if you do that please be sure to import the higher Directory.Bulid.props in that file).

Reply:
keyvault does that for example https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/keyvault/Directory.Build.props.

Reply:
@weshaggard Thanks for the tip, but I don't think that would work in this case since our management plane and data plane SDKs are versioned differently.

Reply:
Not a huge deal to change but we do have a way to condition the properties for mgmt vs data-plane. You could condition it with `$(IsDataPlane)` see https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/keyvault/Directory.Build.props#L2.

Reply:
Working on this in a separate PR. Thanks for the suggestion!